src/Action/GetShortObjectDescriptionAction.php

```diff
@@ -68,7 +69,7 @@ public function __invoke(Request $request)
 
         $object = $admin->getObject($objectId);
 
-        if (!$object && 'html' == $request->get('_format')) {
+        if (!$object && in_array($request->get('_format'), ['html', 'json'])) {
```

Comment:
Why did you add `json` here?

Reply:
Got it now that I expanded whole code.

Reply:
I did not knew if always return empty response when object is not found or just in html or json format.

Reply:
I you want I'll change it to return always an empty response. Or perhaps an 404.

Reply:
Does it throw an exception if `$object` is null and the format is `json` before your change?

Reply:
I just checked and it shouldn't throw an exception, but it will return empty results as JSON, please revert this line.

Reply:
I thought  it would make no sens to return
{
   "id": "",
   "label": ""
}

Reply:
Change it now is a BC Break, but I agree that it makes no sense, cc @sonata-project/contributors

Reply:
ok, I'll change it to return it this way